pkg/cmd/root/pullrequest/create_pullrequest.go

```diff
@@ -168,18 +168,19 @@ func (o *CreatePullRequestOptions) Run() error {
 			GitToken:     o.GitToken,
 			ScmClient:    o.ScmClient,
 		},
-		Gitter:        o.GitClient,
-		CommandRunner: o.CommandRunner,
-		GitKind:       o.GitKind,
-		OutDir:        "",
-		Function:      nil,
-		Labels:        o.Labels,
-		BranchName:    "",
-		ScmClient:     o.ScmClient,
-		BatchMode:     o.BatchMode,
-		Fork:          o.Fork,
-		CommitTitle:   o.Title,
-		CommitMessage: o.Body,
+		Gitter:         o.GitClient,
+		CommandRunner:  o.CommandRunner,
+		GitKind:        o.GitKind,
+		OutDir:         "",
+		Function:       nil,
+		Labels:         o.Labels,
+		BaseBranchName: o.Base,
```

Comment:
We can move this to a separate PR also (including the tests ...) ...

Reply:
Maybe it's my wording in the help for --base that is confusing. When I write HEAD branch I mean the branch that is show has HEAD branch when doing `git remote show origin`. As far as I can see this is the branch the code defaults to when BaseBranchName isn't set. And since main is increasingly popular as the default branch that should work better than master. But maybe the help text should say default branch instead of HEAD branch.

Reply:
I have now modified the wording in the usage text